jetty-servlet/src/test/java/org/eclipse/jetty/servlet/DispatcherTest.java

```diff
@@ -835,13 +836,13 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
             assertEquals(null, request.getPathTranslated());
 
             UrlEncoded query = new UrlEncoded();
```

Comment:
Shouldn't this now be?
``` java
MultiMap<String> query = new MultiMap<>();
```

How did this compile with `new UrlEncode()` now being private?